src/App.tsx

```diff
@@ -1,109 +1,74 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
```

Comment:
Verify that the `Pagination` component is correctly implemented and imported from `./components/Pagination`. It should handle the props for pagination and render the pagination controls correctly.